week03/code/SetsAndMaps_Tests.cs

```diff
@@ -65,36 +66,41 @@ public void FindPairs_NoPairs()
         Assert.AreEqual(Canonicalize(expected), Canonicalize(actual));
     }
 
-    // If this test takes longer than 5 seconds to run, your code is too inefficient.
-    //  On my machine, this executes in ~200 ms with an efficient implementation.
-    [TestMethod, Timeout(5000)]
+    [TestMethod, Timeout(60_000)]
```

Comment:
Keep the timeouts but make them a minute. That way tests will still fail if they are taking way too long.